temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -971,6 +991,8 @@ def __init__(self, *args, **kwargs) -> None:
             _trace("__init__ unrecognized with args %s", args)
 
     def __getattribute__(self, __name: str) -> Any:
+        if HAVE_PYDANTIC and __name == "__get_pydantic_core_schema__":
+            return object.__getattribute__(self, "__get_pydantic_core_schema__")
```

Comment:
Hrmm, trying to think if we even want this generic. I guess all ways to do it would require that the contrib library is at least imported by the user somewhere in the process, but maybe that is ok? If that is ok, I wonder if a side-effect of importing the contrib is ideal over altering the sandbox even if `pydantic` is even available? Though I admit I kinda like the idea of this patch if `pydantic` is even available regardless of contrib lib usage, so I wonder if this should remain.

Reply:
I did wonder about this a bit. I don't see much downside with the way it is currently.